include/matx_tensor_utils.h

```diff
@@ -253,23 +253,8 @@ namespace detail {
         return i(args...);
       }, sliced_tup);
     }
-
-    if constexpr (!(T::Rank() == sizeof...(Is))) {
```

Comment:
I'm pretty sure this was only fixed in 11.5, so we can't take it out if we want 11.4 support